### PR TITLE
1.3.3 added

### PR DESCRIPTION
**Pull request by aamirafridi**

### CODE REVIEW

**Comment by aamirafridi** [comment]
@jimaek can this be merged before it starts conflicting please?

File: files/jquery.marquee/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "jQuery.Marquee",
+  "repo": "aamirafridi/jQuery.Marquee",
+  "files": {
+    "include": ["*.js"]
```
Comment (jimaek) [inline files/jquery.marquee/update.json]: Use `*.min.js` here

Comment (aamirafridi) [inline files/jquery.marquee/update.json]: Ops. Thanks for pointing this out. Done

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**